app/views.py

```diff
@@ -107,33 +98,25 @@ def handle_app(appname):
                 return redir
 
             for app in data['apps']:
-                if appname.lower() == app['name'].lower():
+                if appname.lower() == app['name'].lower() or appname.lower() == app['altname'].lower():
                     app_config = app['config']
 
                     for config in app_config:
 
-                        if config.get('s3filepath'):
```

Comment:
Is there a reason we'd want to remove this? Are we not supporting s3filepath in the config file anymore?